calculator/src/cli.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/local/bin/python3
```

Comment:
This should be:
```
#!/usr/bin/env python3
```
see [here](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take) for a discussion

Reply:
👍